src/shogun/io/SGIO.h

```diff
@@ -288,6 +288,15 @@ class SGIO
 			return location_info;
 		}
 
+		/** get last_progress
```

Comment:
Sorry for being picky, but these things are never fixed otherwise.
You can just remove the text here, and only have the @ in a single line